### PR TITLE
Migrate release pipeline to governed template

### PR DESCRIPTION
**Pull request by LSDima**
Update release pipeline to follow 1ES Pipeline Template.

### CODE REVIEW

**Comment by LSDima** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: .azure-devops/nova-facade-release.yml
Comment (pavelglac) [inline .azure-devops/nova-facade-release.yml:1]: nitpick: compliance-steps.yml is not used and can be deleted.

File: .azure-devops/nova-facade-release.yml [hunk 1]
```diff
@@ -14,52 +21,71 @@ variables:
...
+          Build.SDL.<ESLint>.Enabled: true
+        jobs:
+          - job: compliance
+            displayName: Compliance checks
+            pool:
+              name: Azure-Pipelines-1ESPT-ExDShared
+              image: windows-2022
+              os: windows
+            steps:
+              - template: .azure-devops/steps/service-tree.yml@self
```
Comment (pavelglac) [inline .azure-devops/nova-facade-release.yml:55]: question: isnt this something which is done out of the box in official template? In vscode I am getting warning: 
image

File: .azure-devops/nova-facade-release.yml [hunk 2]
```diff
@@ -14,52 +21,71 @@ variables:
...
-      - template: ./steps/compliance-steps.yml
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: Azure-Pipelines-1ESPT-ExDShared
+        image: windows-2022
+        os: windows
```
Comment (pavelglac) [inline .azure-devops/nova-facade-release.yml:36]: nitpick: you can define pool for our custome jobs and do not need to define it per job
```suggestion
  parameters:
    pool: 
        name: Azure-Pipelines-1ESPT-ExDShared
        image: ubuntu-latest
        os: linux
    sdl:
      sourceAnalysisPool:
        name: Azure-Pipelines-1ESPT-ExDShared
        image: windows-2022
        os: windows
```

File: .azure-devops/nova-facade-release.yml (hunk 1 above)
Comment (pavelglac) [reply .azure-devops/nova-facade-release.yml:55]: After remove it. I has slighly different folder structure in SBom-X. 
image

File: .azure-devops/nova-facade-release.yml
Comment (LSDima) [reply .azure-devops/nova-facade-release.yml:1]: Removed.

File: .azure-devops/nova-facade-release.yml (hunk 1 above)
Comment (LSDima) [reply .azure-devops/nova-facade-release.yml]: After discussing with Mikhail we've figured out the manifest it's not necessary for publishing to npm, so as long as the pipeline it's compliant, we should be good.

File: .azure-devops/nova-facade-release.yml (hunk 2 above)
Comment (LSDima) [reply .azure-devops/nova-facade-release.yml:36]: We only have one job now so I think the outcome it's the same. Previously we've had different pools for different jobs ( compliance - windows / release - linux )

**Review by pavelglac — APPROVED** [review]
LGTM

**Pull request merged**